src/entities/ValueNode/types/ValueSerialized.ts

```diff
@@ -0,0 +1,6 @@
+import { BlockChildType } from '../../BlockNode/types';
+
+/**
+ * Type representing serialized ValueNode
```

Comment:
It's better to describe what actually means to be a value node serialized. Eg. "Serialized ValueNode presented via JSON with the $t property which is used to determine that this structure is a ValueNode"